### ISSUE TITLE
Camera console (and SecurEye application) will tgui bluescreen if you type 'ch' into the search bar

### ISSUE THREAD

**Issue by GPeckman** [tgstation/tgstation#78825]
Reporting client version: 514.1589

## Round ID:
[216216](https://scrubby.melonmesa.com/round/216216)

## Testmerges:
- [Dogs now flee from mobs with tongs](https://www.github.com/tgstation/tgstation/pull/78797)

## Reproduction:

I grabbed a laptop, and put SecurEye on it. Later, I opened it and started typing 'Chief' into the search bar. It blue screened after the second letter. I tried the camera console; same thing happened.

Manually scrolling to and selecting a particular camera entry seemed to work fine, however.

**Comment by SyncIt21** [comment]
@jlsnow301 maybe you can snag this since you have experience with #78412?

**Comment by jlsnow301** [comment]
> @jlsnow301 maybe you can snag this since you have experience with #78412?

i'll take a look at it. 

> It blue screened after the second letter. I tried the camera console; same thing happened.

do you have a screenshot by chance? i'll try to repro this locally, either way.

**Comment by GPeckman** [comment]
I did not catch a screen shot. Even if I had, the most of the error message was blocked by a white square, like the one that happens when a ui hasn't finished loading.

**Comment by GPeckman** [comment]
It happened again, this time I got a screenshot of it: 
![error](https://github.com/tgstation/tgstation/assets/21979502/58cef957-f80f-42a3-8587-e2ce4fd04522)

But like I said, there's the square obscuring most of it. Moving the window around didn't help, the square moved with it.

**Comment by jlsnow301** [comment]
> But like I said, there's the square obscuring most of it. Moving the window around didn't help, the square moved with it.

are you on a standard english keyboard?

**Comment by GPeckman** [comment]
Yes, standard keyboard.

**Comment by SyncIt21** [comment]
@jlsnow301 I feel like #78822 may have something to do with this. It changed the search term from a name to a ref
![Screenshot_20231009-122832_Chrome](https://github.com/tgstation/tgstation/assets/110812394/f7c17ca4-25f2-48f2-97ca-db4b071b6be3)
So maybe typing 'ch' could have confused it ?

**Comment by jlsnow301** [comment]
Good catch, that would probably break search, but I don't think it'd bluescreen from that
I have #78825 up, which might also take care of it

**Comment by jlsnow301** [comment]
ran into this bug on sybil today, it bluescreened once then was fine. assuming my PR did not fix it. strange!

**Comment by GPeckman** [comment]
I actually got a bluescreen from clicking a specific camera (as opposed to bluescreening while typing in the search) as well earlier today. I searched "Warden," only result came up (the Warden's office), and when I clicked it it bluescreened. It looked pretty similar to the screenshot I posted above, complete with the black square blocking most of it.

When I reloaded the gui and scrolled to that camera manually, it didn't bluescreen but the camera itself was nonfunctional. I don't know if that played a role. This happened with a camera console, not SecurEye, and was on Icebox, if that matters.
